src/components/Settings/Export/Export.helpers.js

```diff
@@ -409,7 +435,42 @@ const getDisplaySettings = () => {
   return displaySettings;
 };
 
-export async function openboardExportAdapter(boards = [], intl) {
+/**
+ * Export one or several boards in the Open Board Format. If we specifically
+ * want to export a single board, we generate a single OBF file, otherwise
+ * we generate an OBZ archive.
+ *
+ * @param boardOrBoards A board, or an array of boards.
+ * @param intl
+ * @returns {Promise<void>} Nothing.
+ */
+export async function openboardExportAdapter(boardOrBoards, intl) {
+  return _.isArray(boardOrBoards)
+    ? openboardExportManyAdapter(boardOrBoards, intl)
+    : openboardExportOneAdapter(boardOrBoards, intl);
+}
+
+export async function openboardExportOneAdapter(board, intl) {
+  const { obf } = await boardToOBF({ [board.id]: board }, board, intl, {
+    embed: true
+  });
+  const content = new Blob([JSON.stringify(obf, null, 2)], {
+    type: 'application/json'
+  });
+
+  if (content) {
+    // TODO: Remove illegal characters from the board name.
+    const prefix = moment().format('YYYY-MM-DD_HH-mm-ss-') + board.name + ' ';
```

Comment:
I like the naming convention, anyway we should also modify it for the other adapters (pdf and cboard ).